### DISCUSSION TITLE
I cannot find a way to compare 4 or more files

### DISCUSSION THREAD

**Discussion by Temepest74** [WinMerge/winmerge#1214] [Q&A] [answered]
How to compare more than 3 files? It is not supported? If so, can you give me an alternative?

**Comment by sdottaka** [comment, marked as answer]
Unfortunately, WinMerge can compare only two or three files.

vim/gvim can compare 4 files.

![image](https://user-images.githubusercontent.com/98126/155876145-3dd65b1e-2f73-4a5e-a030-2253dd18d83a.png)
